eZ/Publish/Core/Persistence/Legacy/Content/Type/Handler.php

```diff
@@ -401,8 +401,17 @@ public function copy($userId, $contentTypeId, $status)
         $createStruct->modifierId = $userId;
         $createStruct->created = $createStruct->modified = time();
         $createStruct->creatorId = $userId;
-        $createStruct->remoteId = substr(sha1(uniqid(get_class($createStruct), true)), 0, 7);
-        $createStruct->identifier = 'cpy_' . $createStruct->identifier . '_' . $createStruct->remoteId;
+        $originalRemoteId = $createStruct->remoteId;
+        // truncate remoteId to 32 chars to keep BC length of that field
+        $createStruct->remoteId = substr(sha1(uniqid(get_class($createStruct), true)), 0, 32);
```

Comment:
I used `sha1` because I was actually worried about collisions of truncated `md5` to 7 chars in an identifier of a copy. There's a theoretical possibility of collision but unlikely (we use `sha1` in git commit ids and everybody seems to be fine with hashes truncated up to 7 chars ;) ). 
But good point on DB schema BC break. 

@andrerom: So, should I revert it to `md5` and truncate `md5` hashes?